### ISSUE TITLE
logaddexp转换失败

### ISSUE THREAD

**Issue by reklesswang** [Tencent/ncnn#5595]
layer pnnx.Expression not exists or registered

转换完崩溃了。
看了一下崩溃的地方，是在加载ncnn.param这一行
pnnx.Expression          pnnx_expr_6663           0 1 123

看了一下对应pnnx.param
torch.unsqueeze          torch.unsqueeze_1094     1 1 0 123 dim=1 $input=0
nn.Conv2d                encoder_embed.conv.0     1 1 123 124 bias=True dilation=(1,1) groups=1 in_channels=1 kernel_size=(3,3) out_channels=8 padding=(0,1) padding_mode=zeros stride=(1,1) @bias=(8)f32 @weight=(8,1,3,3)f32
pnnx.Expression          pnnx_expr_6663           0 1 125 expr=0.000000e+00
pnnx.Expression          pnnx_expr_6662           1 1 124 126 expr=sub(@0,1.000000e+00)
aten::logaddexp          pnnx_49                  2 1 125 126 127
pnnx.Expression          pnnx_expr_6657           2 1 127 124 128 expr=sub(sub(@0,mul(@1,8.000000e-02)),3.132617e-01)

**Comment by nihui** [comment]
https://github.com/Tencent/ncnn/pull/5598
